src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  justify-content: left;
+  width: mainWidth($day-width, $calendar-padding);
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: $calendar-padding;
+
+  @for $days from 1 through 31 {
+    &__day-#{$days} {
+      border: 1px solid $color-day-border;
+      background-color: $color-day-background;
+      width: $day-width;
+      height: $day-height;
+      align-items: center;
+      display: flex;
+      justify-content: center;
+      font-size: 30px;
+      font-family: Arial, Helvetica, sans-serif;
+      transition: 0.5s;
```

You should specify what will be transitioned:
```suggestion
      transition: transform 0.5s;
```